client/src/document/ingredients/browser-compatibility-table/utils.ts

```diff
@@ -70,3 +70,16 @@ export function hasNoteworthyNotes(support: bcd.SimpleSupportStatement) {
     !support.partial_implementation
   );
 }
+
+export function needsPrefix(support: bcd.SupportStatement | undefined) {
```

Comment:
Maybe `requiresPrefix` like in the title?

```suggestion
export function requiresPrefix(support: bcd.SupportStatement | undefined) {
```